django/db/models/sql/constants.py

```diff
@@ -35,3 +35,8 @@
 # SQL join types.
 INNER = 'INNER JOIN'
 LOUTER = 'LEFT OUTER JOIN'
+
+# Set operators
+UNION = 'union'
+INTERSECT = 'intersect'
+EXCEPT = 'except'
```

Comment:
@felixxm Ah cool, I was looking in `operations.py` and `features.py` and couldn't find anything :D